pyQuARC/code/gcmd_validator.py

```diff
@@ -208,9 +208,10 @@ def validate_instrument_short_long_name_consistency(self, input_keyword):
         """
         Validates GCMD instrument short name and long name consistency
         """
-        return GcmdValidator.validate_recursively(
+        validity, _ = GcmdValidator.validate_recursively(
             self.keywords["instrument"], input_keyword
         )
```

Comment:
`GcmdValidator.validate_recursively` returns a tuple where the first value is the validity and the second value is the incorrect value. Since we need just `validity` in this check, we make sure we return back only that.

This was a bug, where instead of returning just the validity, we were returning the whole tuple. This change fixes it.